README.md

```diff
@@ -136,6 +136,24 @@ BitTorrent) inside the web browser, as of today.
   * OpenSSL (cli)
   * FFmpeg
 
+## Run using Docker
+
+You can quickly get a server running using Docker. You need to have [docker](https://www.docker.com/community-edition) and [docker-compose](https://docs.docker.com/compose/install/) installed.
+
+Also, you need to run a Traefik loadbalancer using Docker Compose. You can start one locally with this [example compose config](https://gist.github.com/djmaze/72f0565715c59712ce191b41d3c377da).
```

Comment:
I would rephrase so as not let the reader misunderstand Træfik as a hard requirement of PeerTube, but just as a requirement in this example configuration.

Reply:
Right, I rephrased it slightly.

For a local dev environment, we could even go completely without a reverse proxy in front. But I like this approach since it is usable almost out-of-the box for production as well.